src/ShowCommand.php

```diff
@@ -23,32 +26,43 @@ public function execute(InputInterface $input, OutputInterface $output) {
 
     public function showTasks(InputInterface $input, OutputInterface $output) {
         $movieName = $input->getArgument('query');
+        $plot = $input->getOption('fullPlot') ? 'full' : 'short';
 
         $client = new Client();
-        $response = $client->request('GET', "http://www.omdbapi.com/?apikey=c3c73e8d&t={$movieName}");
+        $response = $client->request('GET', "http://www.omdbapi.com/?apikey=c3c73e8d&t={$movieName}&plot={$plot}");
```

Comment:
You can take advantage of http_build_query for generate the query params of the URL here. It will be clearer

https://www.php.net/manual/en/function.http-build-query.php